build.sbt

```diff
@@ -329,6 +329,8 @@ lazy val scaldingCore = module("core").settings(
     "com.twitter" %% "bijection-macros" % bijectionVersion,
     "com.twitter" %% "chill" % chillVersion,
     "com.twitter" %% "chill-algebird" % chillVersion,
+    "com.twitter.elephantbird" % "elephant-bird-cascading2" % elephantbirdVersion,
```

Comment:
@johnynek what's your take on our favorite subject of transitive dependencies? If they are allowed we can remove eb entries from scaldingCommons.